metatensor-torch/src/register.cpp

```diff
@@ -484,7 +484,7 @@ TORCH_LIBRARY(metatensor, m) {
             }
         );
 
-    m.def("read_model_metadata(str path) -> str", read_model_metadata);
+    m.def("read_model_metadata(str path)", read_model_metadata);
```

Comment:
doesn't this need a return type?
```suggestion
    m.def("read_model_metadata(str path) -> __torch__.class.<something...>ModelMetadata", read_model_metadata);
```